### DISCUSSION TITLE
API to publish MAC resume to Manfred

### DISCUSSION THREAD

**Discussion by jaguililla** [getmanfred/mac#23] [Q&A] [answered]
Hello and thanks for your initiative to provide a developer friendly CV format!

I'm considering porting my CV to MAC in order to create a profile in Manfred without the need of filling the form (I know I can do this from the Web).

However, I also wanted to know if Manfred exposes any kind of public API for developers to publish their MAC resumes upon changes using GitHub Actions (assuming the JSON file is located here).

Thanks in advance for your help, and keep on the good work!

**Comment by ydarias** [comment, marked as answer]
Hello @jaguililla 

We already have in the roadmap an issue to offer a public API where you could update your profile using a MAC, but it is not ready yet. With that option, you will be able to create an action that updates your profile automatically.

But ...

What we already have is an option to automatically synchronize your Manfred profile with a repo in Github, using a Manfred application. You just need to go to your profile options.

![imagen](https://user-images.githubusercontent.com/948946/183046305-1646bf61-8584-4cf2-915a-e8e8e70648a4.png)

Then select the 'Data management' section and you will see the instructions to link a Github repository to your Manfred account.

![imagen](https://user-images.githubusercontent.com/948946/183047088-0b8b579c-d349-43ab-87ee-40a932815cc3.png)

This is a bidirectional synchronization, so any change in Manfred will trigger the synchronization in Github and vice-versa.

P.S: Because the MAC is a generalization and Manfred applied some rules, e.g. limiting the soft skills to 3, you can expect some data treatment when you import the data from a MAC into Manfred. Any feedback will be welcome if you believe it can be improved.

**Reply by jaguililla** [reply]
Thanks for the information!
